charts/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Will this LICENSE file be used as part of the Helm Hub entry? If not, the LICENSE file found in the root of this repo might suffice unto its own.

Reply:
Yes, I believe we can remove this.